pimd/pim_nb_config.c

```diff
@@ -2716,11 +2716,11 @@ int lib_interface_igmp_igmp_enable_modify(struct nb_cb_modify_args *args)
 	switch (args->event) {
 	case NB_EV_VALIDATE:
 		if_dnode = yang_dnode_get_parent(args->dnode, "interface");
-		ifp_name = yang_dnode_get_string(if_dnode, ".");
 		mcast_if_count =
 			yang_get_list_elements_count(if_dnode);
 		/* Limiting mcast interfaces to number of VIFs */
 		if (mcast_if_count == MAXVIFS) {
+			ifp_name = yang_dnode_get_string(if_dnode, "name");
```

Comment:
1. It will work, but we use `./` for relative paths throughout the code, so could you, please, change it to `./name`?
2. There are more such issues in this file on lines 1184, 2994, could you fix them too?

Reply:
I certainly can make this change. There is a cost to doing things this way though (the libyang expression evaluation creates multiple tokens etc for "." "/" and "name" vs one token and then evaluates the tokens against the tree. So, is this simply an arbitrarily chosen style? If so, perhaps we should choose the more efficient one. :)

Reply:
Good catch! Perhaps we need to change this.
I'm ok with `"name"` then, but, please, fix two more similar issues.